source/Coop.Core/Client/Services/MobileParties/Messages/SettlementEnterRequest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Common.Messaging;
+using ProtoBuf;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Coop.Core.Client.Services.MapEvents.Messages
+{
+    /// <summary>
+    /// Request entry to a settlement.
+    /// </summary>
+    [ProtoContract(SkipConstructor = true)]
+    public record SettlementEnterRequest : ICommand
```

Comment:
For any network events, I've been prefixing with `Network`
```suggestion
    public record NetworkSettlementEnterRequest : ICommand
```